api/types/constants.go

```diff
@@ -341,6 +341,9 @@ const (
 	// KindHeadlessAuthentication is a headless authentication resource.
 	KindHeadlessAuthentication = "headless_authentication"
 
+	// KindAssistant is a Teleport Assist resource kind.
```

Comment:
Nit: I'd clarify in the comment which assistant-related objects it governs. Conversations/messages/both?

Reply:
Added